google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -379,7 +411,11 @@ class Document:
         pages (List[Page]):
             A list of `Pages` in the `Document`.
         entities (List[Entity]):
-            A list of `Entities` in the `Document`.
+            A list of un-nested `Entities` in the `Document`.
+        chunks (List[documentai.Document.ChunkedDocument.Chunk]):
+            A list of document chunks extracted from a Layout Parser.
+        document_layout_blocks (List[documentai.Document.DocumentLayout.DocumentLayoutBlock]):
+            A list of document layout blocks extracted from a Layout Parser.
```

Comment:
Flattening the graph (tree?) of layout blocks loses the parent-child relation.  This is fine if we are pretty sure this relation is not useful to the users.

Reply:
I agree that it loses the parent-child relation. However, this is still preserved in the `documentai_document`.

It was specifically requested by users to flatten the structure to make it easier to get the details.